contents/tutorials/android-ab-tests.md

```diff
@@ -0,0 +1,273 @@
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.navigation.NavController
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+import com.example.my_ab_test.ui.theme.MyABTestTheme
+
+class MainActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            MyABTestTheme {
+                Surface(modifier = Modifier.fillMaxSize(), color = MaterialTheme.colorScheme.background) {
+                    MyApp()
+                }
+            }
+        }
+    }
+}
+
+@Composable
+fun MyApp() {
+    val navController = rememberNavController()
+    NavHost(navController = navController, startDestination = "main_screen") {
+        composable("main_screen") {
+            Greeting("Android", navController)
+        }
+        composable("second_screen") {
+            SecondScreen()
+        }
+    }
+}
+
+@Composable
+fun Greeting(name: String, navController: NavController, modifier: Modifier = Modifier) {
+    Column(modifier = modifier.fillMaxSize()) {
+        Text(text = "Hello $name!")
+        Button(onClick = { navController.navigate("second_screen") }) {
+            Text("Go to next screen")
+        }
+    }
+}
+
+@Composable
+fun SecondScreen() {
+    val isTestVariant = remember { mutableStateOf(false) } // We'll set this value later in the tutorial
+
+    val backgroundColor = if (isTestVariant.value) Color.Green else Color.Red
+
+    Surface(modifier = Modifier.fillMaxSize(), color = backgroundColor) {
+
+        Column(modifier = Modifier.fillMaxSize(), horizontalAlignment = Alignment.CenterHorizontally, verticalArrangement = Arrangement.Center) {
+            Button(onClick = { /* Do nothing for now */ }) {
+                Text("Click Me!")
+            }
+        }
+    }
+}
+
+@Preview(showBackground = true)
+@Composable
+fun DefaultPreview() {
+    MyABTestTheme {
+        MyApp()
+    }
+}
+```
+
+Make sure to add `implementation("androidx.navigation:navigation-compose:2.4.0")` to your dependencies in to `build.gradle (Module: app)`.
+
+Our basic set up is now complete. Build and run your app to test that it's working.
+
+![Basic setup of the Android app](../images/tutorials/android-ab-tests/android-basic-setup.mp4)
+
+## Adding PostHog to your Android app
+
+First, add the [PostHog Android SDK](/docs/libraries/android) as a dependency in your `build.gradle` file. You can find the latest version on our [GitHub](https://github.com/PostHog/posthog-android/blob/main/CHANGELOG.md). For this tutorial, we use version `2.0.3`.
```

Comment:
```suggestion
First, add the [PostHog Android SDK](/docs/libraries/android) as a dependency in your `app/build.gradle.kts` file. You can find the latest version on our [GitHub](https://github.com/PostHog/posthog-android/blob/main/CHANGELOG.md). For this tutorial, we use version `2.0.3`.
```

Since there are multiple build files and this is the build file form the app itself.

Reply:
+1